examples/shared/envoy/Dockerfile

```
@@ -13,6 +13,12 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     && apt-get -qq update -y \
     && apt-get -qq install --no-install-recommends -y curl
 COPY --chmod=777 "$ENVOY_CONFIG" /etc/envoy.yaml
+HEALTHCHECK \
+    --interval=1s \
+    --timeout=1s \
+    --start-period=1s \
+    --retries=3 \
+    CMD /bin/bash -c "test $(curl -s localhost:8001/stats?filter=server.state | cut -d ':' -f2 | sed 's/ //') == 0 && test $(curl -s localhost:8001/stats?filter=listener_manager.workers_started | cut -d ':' -f2 | sed -s 's/ //') == 1"
```

im wondering if

```suggestion
    CMD curl -s localhost:8001/stats?filter=server.state | grep 0 \
             && curl -s localhost:8001/stats?filter=listener_manager.workers_started | grep 1
```

might be simpler

---

i like it.